### DISCUSSION TITLE
Rancher Kubernetes Local Install (Self-hosted) - User Not Found

### DISCUSSION THREAD

**Discussion by dalemccrory** [Budibase/budibase#5165] [Help]
Hello, 
I'm installing Budibase using Rancher and Kubernetes. The instructions work fine except, when I create the admin user and go to the login screen it shows "User Not Found" after logging in.

Platform: Mac
Rancher: 1.2.1
Kubernetes: 1.23.4
Runtime: dockerd

 Ports forwarding
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/548043/160724332-1cadf8ad-79a1-4049-8ce1-72d9847768be.png">

The ingress doesn't give an IP Address, but I'm able to get the new user screen using the forwarded port
<img width="787" alt="image" src="https://user-images.githubusercontent.com/548043/160724511-279856a3-8359-4a2d-a918-af9669562eb2.png">

Here is what happens after I try logging in for the first time:
<img width="787" alt="image" src="https://user-images.githubusercontent.com/548043/160724786-7bef860d-57fb-4aeb-bb90-245b774c8e4b.png">

Any suggestions on what to try next? Really excited about seeing it up and running...

**Comment by melohagan** [comment]
Hello @dalemccrory 

I was able to re-create your issue, and it turns out there is a bug. 
The CouchDB version used by the helm chart is out of date. 
This will be upgraded in the next release on **Monday 4th April**, so you will be able to try again then.

On release day, I recommend uninstalling budibase: 
```
helm uninstall budibase --namespace budibase
```
And then following through the installation steps again in the documentation: https://docs.budibase.com/docs/kubernetes-k8s

In the meantime, if you're excited to get started then you can get setup with docker-compose! :)
https://docs.budibase.com/docs/docker-compose

**Reply by dalemccrory** [reply]
@melohagan Thanks for finding that bug and giving me insight as to when it will be fixed.

How did you figure out what the issue was? My first thought was "I am going to go look at the logs and see if I can figure it out", but I couldn't figure out how to get the logs and which logs would help me figure out what is going on :)

**Reply by melohagan** [reply]
You're welcome. Finding the bug was quite tricky; lots of trial and error! Got help from @mike12345567 as well, and he raised the issue here: https://github.com/Budibase/budibase/issues/5185
